README.md

```diff
@@ -1,6 +1,6 @@
 # cstor-operators
 [![Go](https://github.com/openebs/cstor-operators/workflows/Go/badge.svg)](https://github.com/openebs/cstor-operators/actions)
```

Comment:
What is this badge for?

Reply:
cc: @shubham14bajpai

Reply:
Go report

Reply:
Okay not for go report but earlier we had an action named go. I will remove this.